core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -215,17 +224,11 @@ class ContainerPool(
         }
         if (positionInLine > available) { // e.g. if there is 1 available, then I wait if I am second in line (positionInLine = 2)
```

Comment:
please fix the comments `// e.g. ...` i think these moved around and now are disconnected and confusing.